reV/SAM/econ.py

```diff
@@ -509,19 +506,15 @@ def _windbos(inputs):
                         outputs = wb.output
         return inputs, outputs
 
-    @property
-    def default(self):
+    @staticmethod
```

Comment:
I'm confused by this change, Is default now just a place holder?

Reply:
Placeholder? no, each concrete reV-SAM object will have a default() method to retrieve the default object. Our `Sam` wrapper has an abstractmethod placeholder but all the concrete classes will have useful methods.

Previously it was a property which was kind of nice and natural to call, but the problem was that you had to instantiate the object before getting the default object. I think a static method is much better, because now we can explicitly require the resource, meta, and config inputs separately from a default object.